exercises/phone-number/canonical-data.json

```diff
@@ -72,15 +72,37 @@
         },
         {
           "description": "invalid if area code does not start with 2-9",
-          "property": "clean",
-          "phrase": "(123) 456-7890",
-          "expected": null
+          "cases": [
+            {
+              "description": "invalid if area code starts with 0",
+              "property": "clean",
+              "phrase": "(023) 456-7890",
+              "expected": null
+            },
+            {
+              "description": "invalid if area code starts with 1",
+              "property": "clean",
+              "phrase": "(123) 456-7890",
+              "expected": null
+            }
+          ]
         },
```

Comment:
Agree with the new edge test cases.  But, would prefer it without another level of "cases": [ ].  I suggest the two invalid cases be at same level as the other clean test cases.  Otherwise, the change will break some generators unnecessarily.

Reply:
Fair enough; done.

Reply:
Thanks!